templates/CRM/common/civicrm.settings.php.template

```diff
@@ -420,6 +420,13 @@ if (!defined('CIVICRM_DB_CACHE_PREFIX')) {
 // define('CIVICRM_GETTEXT_NATIVE', 1);
 // }
 
+/**
+ * Define how many times to retry a transaction when the DB hits a deadlock
+ * (ie. the database is locked by another transaction). This is an
+ * advanced setting intended for high-traffic databases & experienced developers/ admins.
+ */
+define('CIVICRM_DEADLOCK_RETRIES', 3);
```

Comment:
Is this adding the define in when it wasn't already added?  Or was it defined/defaulted elsewhere?

Reply:
it's new - but requires the merge in packages too